plugins/action/k8s_info.py

```diff
@@ -26,6 +25,32 @@
 from ansible.plugins.action import ActionBase
 
 
+class RemoveOmit(object):
+    def __init__(self, buffer, omit_value):
+        try:
+            import yaml
+        except ImportError:
+            raise AnsibleError("Failed to import the required Python library (PyYAML).")
+        self.data = yaml.safe_load_all(buffer)
+        self.omit = omit_value
+
+    def remove_omit(self, data):
+        if isinstance(data, dict):
+            result = dict()
+            for key, value in iteritems(data):
+                if value == self.omit:
+                    continue
+                result[key] = self.remove_omit(value)
+            return result
+        elif isinstance(data, list):
+            return [self.remove_omit(v) for v in data if v != self.omit]
+        else:
+            return data
```

Comment:
Can this code be written as following - 

```suggestion
        if isinstance(data, dict):
            result = dict()
            for key, value in iteritems(data):
                if value == self.omit:
                    continue
                result[key] = self.remove_omit(value)
            return result
        if isinstance(data, list):
            return [self.remove_omit(v) for v in data if v != self.omit]
        return data
```